.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,28 @@
 <!--- Provide a general summary of your changes in the Title above -->
+
 ## Description
 <!--- Describe your changes in detail -->
+
 ## Related Issue
-<!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
-<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
+<!--- If there's an issue related, please link it here -->
+<!--- If suggesting a new feature or a medium/big change, please discuss it in an issue first. -->
+<!--- For small changes, it's okay to open a PR immediately. -->
+
 ## Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
+<!--- Why is this change required? What problem does it solve? What does it improve? -->
+
 ## How Has This Been Tested?
 <!--- Please describe in detail how you tested your changes. -->
 <!--- Include details of your testing environment, and the tests you ran to -->
 <!--- see how your change affects other areas of the code, etc. -->
-## Screenshots (if appropriate):
-## Types of changes
-<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+<!--- If your changes are tested by the CI, you can just write that.-->
+
+## Screenshots
+<!--- (if appropriate) -->
+
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My code follows the code style of this project.
-- [ ] I have updated the relevant documentation.
 - [ ] I have read the **CONTRIBUTING** document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+- [ ] I have updated the relevant documentation.
+- [ ] I have added tests for new logic.
```

Comment:
```suggestion
- [ ] I have added tests for the new logic.
```

Reply:
Actually, that was my intention.

"I have added tests for the new logic" IIUC implies "there *is* the new logic, and I have added tests for it" which is not necessarily true, if my PR doesn't add any new logic.
"I have added tests for new logic" is "for new logic, I have added tests" and if there's no new logic, then obv I added nothing.